src/Raven.Server/Documents/PeriodicBackup/RavenFtpClient.cs

```diff
@@ -65,135 +67,121 @@ public void UploadFile(string folderName, string fileName, Stream stream)
             var url = CreateNestedFoldersIfNeeded(folderName);
             url += $"/{fileName}";
 
-            var request = CreateFtpWebRequest(url, WebRequestMethods.Ftp.UploadFile, keepAlive: true);
-            var readBuffer = new byte[DefaultBufferSize];
-
-            int count;
-            var requestStream = request.GetRequestStream();
-            while ((count = stream.Read(readBuffer, 0, readBuffer.Length)) != 0)
+            using (var client = CreateFtpClient(_url, keepAlive: true))
```

Comment:
we should add error handling to this method, check other clients, how they handle uploads, how the handle progress state change